package/run.sh

```diff
@@ -54,7 +54,7 @@ echo "Rancher: Running CIS Benchmarks"
 rm -rf "${SONOBUOY_OUTPUT_DIR}"/*.tar.gz
 
 # Run sonobuoy first
-if ! sonobuoy master -v 3
+if ! sonobuoy aggregator -v 3
```

Comment:
Did the sonobuoy command change?

Reply:
yes there was a pr to remove insensitive terms https://github.com/vmware-tanzu/sonobuoy/commit/046c0827bd950c5c0117812fb76c9854edea5279

Reply:
ok, thanks